pyproject.toml

```diff
@@ -53,6 +53,7 @@ aleph-alpha-client = {version="^2.15.0", optional = true}
 deeplake = {version = "^3.2.9", optional = true}
 pgvector = {version = "^0.1.6", optional = true}
 psycopg2-binary = {version = "^2.9.5", optional = true}
+boto3 = "^1.26.96"
```

Comment:
will likely not want to add this as required

Reply:
Yes, that was only added to support DynamoDB, but we can leave it for the user to install it.